src/settings/Settings.cpp

```diff
@@ -838,6 +861,12 @@ void Settings::setRenamings(RenameType renameType, bool files, bool folders, boo
     settings()->setValue(QString("RenamePattern/%1/RenameFolders").arg(renameTypeStr), folders);
     settings()->setValue(QString("RenamePattern/%1/UseSeasonDirectories").arg(renameTypeStr), seasonDirectories);
 }
+void Settings::setRenamings(RenameType renameType, bool files, bool folders, bool seasonDirectories, bool replaceDelimiter)
+{
+    Settings::setRenamings(renameType, files, folders, seasonDirectories);
+    const QString renameTypeStr = renamerTypeToString(renameType);
+    settings()->setValue(QString("RenamePattern/%1/ReplaceDelimiter").arg(renameTypeStr), replaceDelimiter);
```

Comment:
I'd prefer `Delimiter`, because that's what it is intended to be: a delimiter between words. That we only replace spaces is a restriction or implementation detail.

Reply:
Ah, this comment was meant for the string value, not the boolean value. :)